Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/SimplifiedLogin/SimplifiedLoginManagerTests.swift

```diff
@@ -0,0 +1,60 @@
+import XCTest
+import Cuckoo
+@testable import AccountSDKIOSWeb
+
+final class SimplifiedLoginManagerTests: XCTestCase {
+ 
+    // MARK: -
+    
+    func testGetSimplifiedLogin_noLoggedInSessionInSharedKeychain (){
+        let mockHTTPClient = MockHTTPClient()
+        let client = Client(configuration: Fixtures.clientConfig, httpClient: mockHTTPClient)
+        let sut = SimplifiedLoginManager(client: client, contextProvider: ASWebAuthSessionContextProvider(), env: .pre, completion: { result in })
+        
+        let expectation = self.expectation(description: "Should fail with SimplifiedLoginError.noLoggedInSessionInSharedKeychain")
+        sut.getSimplifiedLogin(nil, completion: { result in
```

Comment:
should be requestSimplifiedLogin (current name on feature/917-simplified-login branch)

Reply:
I guess I was on the pre-rebased branch. It is updated now 👍

Reply:
Works now! but why I didn't see it in PR, I don't know 🤔  (there were no merge conflicts to feature branch)